src/Stage.ts

```diff
@@ -21,14 +20,15 @@ class StageComponent extends Component {
 	protected render(): void {
 		const elements: NodeListOf<HTMLElement> = document.querySelectorAll(this.getTemplate());
 
-		if (elements.length === 0) {
-			this.getLogger().fatal("Invalid CSS seletor pattern provided: " + this.getTemplate());
-			throw new SelectorError("Invalid CSS seletor pattern provided: " + this.getTemplate());
-		}
+		let elength = elements.length;
+		let errmsg = (elength !== 1) ?
+			"CSS selector MUST identify single HTMLElement: %pattern% - %found% found" : null;
 
-		if (elements.length > 1) {
-			this.getLogger().fatal("CSS selector pattern provided is NOT unique: " + this.getTemplate());
-			throw new SelectorError("CSS selector pattern provided is NOT unique: " + this.getTemplate());
+		if(errmsg) {
+			let patSubObj = {'%pattern%': this.getTemplate(), '%found%': elength};
+			let errobj: SelectorError = new SelectorError(errmsg, patSubObj);
```

Comment:
1. use const instead of let
2. Camel case variable name

Reply:
Done.